<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MadMan</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>2</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTabWidget" name="tabWidget_index">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::West</enum>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget_main">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionImport"/>
    <addaction name="actionFormat"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionCloseAll_Main"/>
    <addaction name="separator"/>
    <addaction name="actionCloseAll_Right"/>
    <addaction name="actionTabAll_Right"/>
    <addaction name="actionTwoCol_Right"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionImport"/>
   <addaction name="actionFormat"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
   <addaction name="actionCloseAll_Main"/>
   <addaction name="actionCloseAll_Right"/>
   <addaction name="actionTabAll_Right"/>
   <addaction name="actionTwoCol_Right"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../rc.qrc">
     <normaloff>:/image/Open</normaloff>:/image/Open</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../rc.qrc">
     <normaloff>:/image/Close</normaloff>:/image/Close</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../rc.qrc">
     <normaloff>:/image/Quit</normaloff>:/image/Quit</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionCloseAll_Main">
   <property name="icon">
    <iconset resource="../rc.qrc">
     <normaloff>:/image/CloseAll_Main</normaloff>:/image/CloseAll_Main</iconset>
   </property>
   <property name="text">
    <string>CloseAll [Main]</string>
   </property>
  </action>
  <action name="actionCloseAll_Right">
   <property name="icon">
    <iconset resource="../rc.qrc">
     <normaloff>:/image/CloseAll_Right</normaloff>:/image/CloseAll_Right</iconset>
   </property>
   <property name="text">
    <string>CloseAll [Right]</string>
   </property>
  </action>
  <action name="actionTabAll_Right">
   <property name="icon">
    <iconset resource="../rc.qrc">
     <normaloff>:/image/TabAll_Right</normaloff>:/image/TabAll_Right</iconset>
   </property>
   <property name="text">
    <string>TabAll [Right]</string>
   </property>
  </action>
  <action name="actionTwoCol_Right">
   <property name="icon">
    <iconset resource="../rc.qrc">
     <normaloff>:/image/TwoCol_Right</normaloff>:/image/TwoCol_Right</iconset>
   </property>
   <property name="text">
    <string>TwoCol [Right]</string>
   </property>
  </action>
  <action name="actionFormat">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../rc.qrc">
     <normaloff>:/image/Format</normaloff>:/image/Format</iconset>
   </property>
   <property name="text">
    <string>Format</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../rc.qrc">
     <normaloff>:/image/Import</normaloff>:/image/Import</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>AboutQt</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../rc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCloseAll_Main</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>closeall_main()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCloseAll_Right</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>closeall_right()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTabAll_Right</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>taball_right()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTwoCol_Right</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>twocol_right()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFormat</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>format_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionImport</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>import()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>close_file()</slot>
  <slot>open_file()</slot>
  <slot>closeall_main()</slot>
  <slot>closeall_right()</slot>
  <slot>taball_right()</slot>
  <slot>twocol_right()</slot>
  <slot>format_file()</slot>
  <slot>import()</slot>
 </slots>
</ui>
